client/rest/src/plugins/rosetta/nem/blockRoutes.js

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2016-2019, Jaguar0625, gimre, BloodyRookie, Tech Bureau, Corp.
+ * Copyright (c) 2020-present, Jaguar0625, gimre, BloodyRookie.
+ * All rights reserved.
+ *
+ * This file is part of Catapult.
+ *
+ * Catapult is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Catapult is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Catapult.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { OperationParser } from './OperationParser.js';
+import { createLookupCurrencyFunction, getBlockchainDescriptor } from './rosettaUtils.js';
+import Block from '../openApi/model/Block.js';
+import BlockIdentifier from '../openApi/model/BlockIdentifier.js';
+import BlockRequest from '../openApi/model/BlockRequest.js';
+import BlockResponse from '../openApi/model/BlockResponse.js';
+import BlockTransactionRequest from '../openApi/model/BlockTransactionRequest.js';
+import BlockTransactionResponse from '../openApi/model/BlockTransactionResponse.js';
+import { RosettaErrorFactory, rosettaPostRouteWithNetwork } from '../rosettaUtils.js';
+import { NetworkLocator } from 'symbol-sdk';
+import { Network, NetworkTimestamp } from 'symbol-sdk/nem';
+
+export default {
+	register: (server, db, services) => {
+		const GENESIS_BLOCK_NUMBER = 1;
+
+		const blockchainDescriptor = getBlockchainDescriptor(services.config);
+		const network = NetworkLocator.findByName(Network.NETWORKS, blockchainDescriptor.network);
+		const lookupCurrency = createLookupCurrencyFunction(services.proxy);
+		const parser = new OperationParser(network, {
+			includeFeeOperation: true,
+			operationStatus: 'success',
+			lookupCurrency
+		});
+
+		server.post('/block', rosettaPostRouteWithNetwork(blockchainDescriptor, BlockRequest, async typedRequest => {
+			const height = typedRequest.block_identifier.index;
+			const blockInfo = await services.proxy.fetch('local/block/at', undefined, {
+				method: 'POST',
+				body: JSON.stringify({ height }),
+				headers: { 'Content-Type': 'application/json' }
+			});
+
+			const parseTransaction = tx => parser.parseTransactionAsRosettaTransaction(tx.tx, { hash: { data: tx.hash } });
```

Comment:
tx => transaction

Reply:
any reason not to inline below?

Reply:
It was just to reduce the line length but I will put it on multiple lines